Introduce prototype-style function definitions for the POWER wordcopy
functions.  Upstream, this happened as a side-effect of this commit:

commit ea5a72f882f878a5ea00d69e56a19c2fc3fd2d2a
Author: Adhemerval Zanella <azanella@linux.vnet.ibm.com>
Date:   Mon Oct 14 09:59:59 2013 -0500

    PowerPC: multiarch wordcopy routines for PowerPC32

diff --git a/sysdeps/powerpc/powerpc32/power4/wordcopy.c b/sysdeps/powerpc/powerpc32/power4/wordcopy.c
index e8a35eaeef6e92fe..233a183cd8d8cf8b 100644
--- a/sysdeps/powerpc/powerpc32/power4/wordcopy.c
+++ b/sysdeps/powerpc/powerpc32/power4/wordcopy.c
@@ -27,10 +27,7 @@
    Both SRCP and DSTP should be aligned for memory operations on `op_t's.  */
 
 void
-_wordcopy_fwd_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_fwd_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1;
 
@@ -65,10 +62,7 @@ _wordcopy_fwd_aligned (dstp, srcp, len)
    *not* be aligned.  */
 
 void
-_wordcopy_fwd_dest_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_fwd_dest_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1, a2;
   int sh_1, sh_2;
@@ -119,10 +113,7 @@ _wordcopy_fwd_dest_aligned (dstp, srcp, len)
    operations on `op_t's.  */
 
 void
-_wordcopy_bwd_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_bwd_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1;
 
@@ -158,10 +149,7 @@ _wordcopy_bwd_aligned (dstp, srcp, len)
    operations on `op_t', but SRCP must *not* be aligned.  */
 
 void
-_wordcopy_bwd_dest_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_bwd_dest_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1, a2;
   int sh_1, sh_2;
diff --git a/sysdeps/powerpc/powerpc32/power6/wordcopy.c b/sysdeps/powerpc/powerpc32/power6/wordcopy.c
index 698b82f4fbda9445..bee5e9d53e5fd780 100644
--- a/sysdeps/powerpc/powerpc32/power6/wordcopy.c
+++ b/sysdeps/powerpc/powerpc32/power6/wordcopy.c
@@ -28,10 +28,7 @@
    Both SRCP and DSTP should be aligned for memory operations on `op_t's.  */
 
 void
-_wordcopy_fwd_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_fwd_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1;
 
@@ -80,10 +77,7 @@ _wordcopy_fwd_aligned (dstp, srcp, len)
   while (len != 0)
 
 void
-_wordcopy_fwd_dest_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_fwd_dest_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1, a2;
   int sh_1, sh_2;
@@ -125,10 +119,7 @@ _wordcopy_fwd_dest_aligned (dstp, srcp, len)
    operations on `op_t's.  */
 
 void
-_wordcopy_bwd_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_bwd_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1;
 
@@ -178,10 +169,7 @@ _wordcopy_bwd_aligned (dstp, srcp, len)
    operations on `op_t', but SRCP must *not* be aligned.  */
 
 void
-_wordcopy_bwd_dest_aligned (dstp, srcp, len)
-     long int dstp;
-     long int srcp;
-     size_t len;
+_wordcopy_bwd_dest_aligned (long int dstp, long int srcp, size_t len)
 {
   op_t a0, a1, a2;
   int sh_1, sh_2;
